src/sqlfluff/dialects/dialect_postgres_keywords.py

```diff
@@ -297,6 +298,7 @@ def get_keywords(keyword_list, keyword_type):
     ("EXP", "not-keyword"),
     ("EXPLAIN", "non-reserved"),
     ("EXPRESSION", "non-reserved"),
+    ("EXTENDED", "non-reserved"),
```

Comment:
This list is made to match the official docs (https://www.postgresql.org/docs/14/sql-keywords-appendix.html). Could you move this keyword down to the `postgres_nondocs_keywords` section below?

Reply:
Thanks for this. I now know where to reference what goes where!